internal/ui/ui_glfw.go

```diff
@@ -308,6 +308,10 @@ func getMonitorFromPosition(wx, wy int) *Monitor {
 	return nil
 }
 
+func initialWindowPosition(mw, mh, ww, wh int) (x, y int) {
+	return (mw - ww) / 2, (mh - wh) / 3
+}
```

Comment:
This logic is still duplicated with the ebiten package's `initializeWindowPositionIfNeeded`, right? I'd expose this function and call it from ebiten.

Reply:
Didn't see that. Fixed now.